### ISSUE TITLE
[Bug]: The "net" section in the cgroups container is picking the wrong / random interface name

### ISSUE THREAD

**Issue by sashwathn** [netdata/netdata-cloud#666]
### Bug description

The net section under cgroups (container) adds a random interface to the sub-section name (probably from one of the containers).

### Expected behavior

The sub-section should only be named generically as "net".

### Steps to reproduce

1. Create multiple containers on the system with different virtual interfaces
2. Open the Cloud dashboard and look at the cgroups container section.

### Screenshots

<img width="245" alt="image" src="https://user-images.githubusercontent.com/96257330/203098975-06214466-c17c-414a-8d9d-9b3136d6bf31.png">

For example, the net eth0 section in the image should be named as "net" only - especially for cases where there are multiple virtual interfaces belonging to different containers.

### Desktop

This is agent independent.

**Comment by hugovalente-pm** [comment]
@papazach @juacker should a mutation/transformation be applied on Cloud BE to show `net all` ? I know we something similar on Disks, Network Interfaces
@sashwathn I would suggest probably decreasing this to `medium` prio, this isn't blocking users from using the charts/section. @dimko wdyt?

**Comment by sashwathn** [comment]
@juacker : Can you check if the "net eth0" (or any other interface name) is being sent from the agent as the family or is derived on the backend? I agree completely with you and the family simply needs to be "net" and then the user has an option of filtering on labels.

**Comment by juacker** [comment]
yes @sashwathn the value `net eth0` comes directly from the agent, here are some values:

![image](https://user-images.githubusercontent.com/2930882/203522674-d646b07b-70a4-48eb-89a9-01168ba4809e.png)

**Comment by sashwathn** [comment]
@MrZammler : Can we check this and handle it in general for also other sections that may have a similar behaviour?

**Comment by sashwathn** [comment]
@MrZammler : Can we get some fix for this issue?

**Comment by hugovalente-pm** [comment]
@MrZammler not sure if you had a chance to look at this but for now moving this to `netdata/netdata` since this is the family coming from the collector
